src/loader/load.c

```diff
@@ -0,0 +1,66 @@
+#include <postgres.h>
+#include <commands/extension.h>
+#include <access/relscan.h>
+#include <catalog/pg_extension.h>
+#include <utils/fmgroids.h>
+#include <utils/builtins.h>
+#include <utils/rel.h>
+#include <catalog/indexing.h>
+
+#include "load.h"
+#include "extension.h"
+
+static char *
+extension_version(void)
```

Comment:
Should this function fail with an error if extension not found? Probably won't happen, but would be good for clarity and declaring that the result should always be valid instead of returning NULL.